### ISSUE TITLE
Theme installation error on Terminator 2.1.2

### ISSUE THREAD

**Issue by salimsket** [EliverLara/terminator-themes#45]
Terminator version: 2.1.2

Getting this error in the logs:
```
Traceback (most recent call last):
  File "/home/USR/.config/terminator/plugins/terminator-themes.py", line 253, in on_install
    self.terminal.config.add_profile(target["name"])
TypeError: Config.add_profile() missing 1 required positional argument: 'toclone'
```

Adding `None` at the end seems to fix it.

`self.terminal.config.add_profile(target["name"], None)`

---

After that, getting next error:
```
Traceback (most recent call last):
  File "/home/USR/.config/terminator/plugins/terminator-themes.py", line 266, in on_install
    self.config_base.set_item(k, v, target["name"])
  File "/usr/lib/python3.10/site-packages/terminatorlib/config.py", line 812, in set_item
    raise KeyError('ConfigBase::set_item: unknown key %s' % key)
KeyError: 'ConfigBase::set_item: unknown key cursor_color'
```

This quick dirty fix seems to work:
![image](https://user-images.githubusercontent.com/38206944/200818841-bfaea226-2248-4dd4-ad4d-37fb3efb7c0c.png)

**Comment by tabcodes** [comment]
FWIW- I tested the fix above with Terminator 2.1.2 on Linux Mint 21, and it worked like a charm. Thank you!

**Comment by EliverLara** [comment]
Thanks for pointing the error it was fixed with [2daa15e](https://github.com/EliverLara/terminator-themes/commit/2daa15e01e6cb3abd6535c2d4b7727b8eaf18b77) :)
